src/parson.c

```diff
@@ -1650,6 +1650,65 @@ JSON_Status json_serialize_to_file_pretty(const JSON_Value *value, const char *f
     return return_code;
 }
 
+JSON_Status json_serialize_to_file_pretty_a(const JSON_Value *value, const char *filename) {
+    JSON_Status return_code = JSONSuccess;
+    FILE *fp = NULL;
+    char *serialized_string = NULL; 
+
+    fp = fopen(filename, "a");
+    if (fp == NULL) {
+        json_free_serialized_string(serialized_string);
```

Comment:
Dont need to free the string here right?

Reply:
correct, will remove